spec/finalization-group.html

```diff
@@ -210,7 +211,9 @@ <h1>%FinalizationGroupCleanupIteratorPrototype%.next ( )</h1>
             1. Let _finalizationGroup_ be _iterator_.[[FinalizationGroup]].
             1. If _finalizationGroup_ is *undefined*, then
```

Comment:
When will this condition be true? I assumed that this was some vestigial logic for the same case (or is vestigial logic for the shutdown method?)

Reply:
In case the iterator is leaked, then the finalization group doesn't need to leak as well. With this condition the finalization group is weakly held and can be collected independent of the iterator.

Reply:
I don't understand what makes it weakly held. Should we update the "abstract jobs" section to indicate that this is weak somehow?

Reply:
Yeah, we should update that too. Both are required.